src/generator.rs

```diff
@@ -76,40 +73,6 @@ pub fn run_generator(
     Ok(())
 }
 
-fn make_service_template(output_directory: &Path) -> Result<()> {
-    let service_path = output_directory.join("swap-create@.service");
-
-    let contents = format!(
-        "\
-# Automatically generated by zram-generator
-
-[Unit]
-Description=Create swap on /dev/%i
-Wants=systemd-modules-load.service
-After=systemd-modules-load.service
-After=dev-%i.device
-DefaultDependencies=false
-
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStartPre=-modprobe zram
-ExecStart={generator} --setup-device '%i'
-",
-        generator = env::current_exe()
-            .context("Couldn't get path to generator executable")?
-            .display(),
-    );
-    fs::write(&service_path, contents).with_context(|| {
-        format!(
-            "Failed to write a device service into {}",
-            service_path.display()
-        )
-    })?;
-
-    Ok(())
-}
-
```

Comment:
Hmm, maybe include a drop-in that overrides ExecStart if the generator is run from a non-standard location? This will make testing a lot easier.